policy/modules/system/systemd-homed.fc

```diff
@@ -0,0 +1,27 @@
+#
+# homed file context
+#
+
+/usr/lib/systemd/systemd-homed                            --  gen_context(system_u:object_r:systemd_homed_exec_t,s0)
+/usr/lib/systemd/systemd-homework                         --  gen_context(system_u:object_r:systemd_homework_exec_t,s0)
+/usr/lib/systemd/system/systemd-homed-activate\.service   --  gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/system/systemd-homed\.service            --  gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+
+/var/home/(.+)\.home                                      --  gen_context(system_u:object_r:systemd_homed_crypto_luks_t,s0)
+/var/home/(.+)\.homedir                                   --  gen_context(system_u:object_r:user_home_dir_t,s0)
```

Comment:
this should go to userdomain.fc

Reply:
These are HOME_ROOT, not HOME. Should I move them to files.fc?

Reply:
you should not use types outside of this modules directly

Reply:
Thank you for the clarification.